content/en/blog/_posts/2020-12-10-Pod Impersonation and Short-lived Volumes in CSI Drivers.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: Pod Impersonation and Short-lived Volumes in CSI Drivers'
+date: 2020-12-10
+slug: kubernetes-release-1.20-Pod-Impersonation-and-Short-lived-Volumes-in-CSI-Drivers
+---
+
+**Author**: Shihang Zhang (Google)
+
+Typically when a CSI driver mounts credentials such as secrets and certificates, it has to authenticate against storage providers to access the credentials. However, the access to those credentials are controlled basis the pods identities rather than the CSI driver's identity. CSI drivers, therefore, need some way to retrieve pod's service account token. 
```

Comment:
"are controlled based upon" or "are controlled on the basis of"?

Reply:
I'd consider expanding CSI or making it a hyperlink to a page about CSI in Kubernetes.

Reply:
@zshihang PTAL at comments by @sftim above. Please suggest if you have any inputs on this.

Reply:
> "are controlled based upon" or "are controlled on the basis of"?

Addressed this with the latest commit, thanks!

Reply:
> I'd consider expanding CSI or making it a hyperlink to a page about CSI in Kubernetes.

@zshihang : PTAL & advise

Reply:
we can add `https://github.com/container-storage-interface/spec/blob/baa71a34651e5ee6cb983b39c03097d7aa384278/spec.md` as the hyperlink for CSI.

Reply:
Addressed this with the latest commit, PTAL @zshihang & @sftim